src/wr_logic_teleop_ds/src/TeleopDriveTrainLogic.cpp

```diff
@@ -129,8 +134,8 @@ int main(int argc, char** argv){
 		
 		//Set the left and right values to be the product of respective raw speeds and speed ratios
 		if(!dog.isMad()){
-			output.left_value = speedRaw[0]*speedRatio[0];
-			output.right_value = speedRaw[1]*speedRatio[1];
+			output.left_value = (abs(speedRaw[0]) < deadband ? 0 : speedRaw[0])*speedRatio[0];
+			output.right_value = (abs(speedRaw[1]) < deadband ? 0 : speedRaw[1])*speedRatio[1];
```

Comment:
This will jump immediately jump the output from 0 to deadband * speedRatio after the joystick crosses the headband. This might be what we want but we should consider something like speedRatio/ (1-deadband) * (speedRaw-deadband). This would scale the output smoothly with the input in exchange for some loss of precision.

Reply:
Also, it might make sense to delegate this change to the wreadinput so its available for all teleop control

Reply:
I agree with your assessment.  These changes will go into `wreadinput` as controller settings.  Thanks for the feedback!